### PR TITLE
Some structure fixes and improvements

### PR DESCRIPTION
**Pull request by elenaspb2019**
Some structure fixes and improvements

Before submitting a pull request, please do the following steps:

1. Read [instructions for contributors](https://catboost.ai/docs/concepts/development-and-contributions.html).
2. Run `ya make` in catboost folder to make sure the code builds.
3. Add tests that test your change.
4. Run tests using `ya make -t -A` command.
5. If you haven't already, complete the [CLA](https://yandex.ru/legal/cla/).

I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en.

### CODE REVIEW

File: catboost/docs/en/features/visualization_jupyter-notebook.md [hunk 1]
```diff
@@ -44,33 +45,34 @@ Refer to the [MetricVisualizer](../concepts/python-reference_catboostipythonwid
...
-loss_function = "CrossEntropy",
-train_dir = "crossentropy")
+   train_labels = [1,1,0]
 
-model.fit(train_data, train_labels, cat_features)
-predictions = model.predict(train_data)
-```
+   model = CatBoostClassifier(iterations=20,
+   loss_function = "CrossEntropy",
+   train_dir = "crossentropy")
```
Comment (Evgueni-Petrov-aka-espetrov) [inline catboost/docs/en/features/visualization_jupyter-notebook.md:62]: please add indentation (4 spaces)

File: catboost/docs/en/features/visualization_jupyter-notebook.md [hunk 2]
```diff
@@ -44,33 +45,34 @@ Refer to the [MetricVisualizer](../concepts/python-reference_catboostipythonwid
...
 
-train_data = [["a", "b", 1, 4, 5, 6],
-["a", "b", 4, 5, 6, 7],
-["c", "d", 30, 40, 50, 60]]
+   cat_features = [0,1,2]
 
-train_labels = [1,1,0]
+   train_data = [["a", "b", 1, 4, 5, 6],
+   ["a", "b", 4, 5, 6, 7],
+   ["c", "d", 30, 40, 50, 60]]
```
Comment (Evgueni-Petrov-aka-espetrov) [inline catboost/docs/en/features/visualization_jupyter-notebook.md:56]: please add indentation (4 spaces)

File: catboost/docs/en/features/visualization_jupyter-notebook.md [hunk 3]
```diff
@@ -92,51 +94,53 @@ Refer to the [MetricVisualizer](../concepts/python-reference_catboostipythonwid
...
 
-     train_data = [["a", "b", 1, 4, 5, 6],
-     ["a", "b", 4, 5, 6, 7],
-     ["c", "d", 30, 40, 50, 60]]
+    cat_features = [0,1,2]
 
-     train_labels = [1,1,0]
+    train_data = [["a", "b", 1, 4, 5, 6],
+    ["a", "b", 4, 5, 6, 7],
+    ["c", "d", 30, 40, 50, 60]]
```
Comment (Evgueni-Petrov-aka-espetrov) [inline catboost/docs/en/features/visualization_jupyter-notebook.md:105]: please add indentation (4 spaces)

**Review by Evgueni-Petrov-aka-espetrov — CHANGES_REQUESTED** [review]

File: catboost/docs/en/features/visualization_jupyter-notebook.md (hunk 3 above)
Comment (elenaspb2019) [reply catboost/docs/en/features/visualization_jupyter-notebook.md]: > please add indentation (4 spaces)

Added

File: catboost/docs/en/features/visualization_jupyter-notebook.md (hunk 2 above)
Comment (elenaspb2019) [reply catboost/docs/en/features/visualization_jupyter-notebook.md]: > please add indentation (4 spaces)

Added

File: catboost/docs/en/features/visualization_jupyter-notebook.md (hunk 1 above)
Comment (elenaspb2019) [reply catboost/docs/en/features/visualization_jupyter-notebook.md]: > please add indentation (4 spaces)

Added

**Review by Evgueni-Petrov-aka-espetrov — APPROVED** [review]

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
CLA already signed

**Comment by arcadia-devtools** [comment]
@Evgueni-Petrov-aka-espetrov, internal review request created: [3196291](https://a.yandex-team.ru/review/3196291)

**Review by Evgueni-Petrov-aka-espetrov — APPROVED** [review]

**Comment by Evgueni-Petrov-aka-espetrov** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
@Evgueni-Petrov-aka-espetrov, internal review request updated: [3196291](https://a.yandex-team.ru/review/3196291)

**Comment by andrey-khropov** [comment]
Auto closing of merged PRs has been broken, so I have to close it manually.

**Pull request closed without merging**